File: app/src/main/java/com/BlizzardArmory/warcraft/account/PlayableClass.java

package com.BlizzardArmory.warcraft.account;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PlayableClass {

    @SerializedName("key")
    @Expose
    private Key_ key;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("id")
    @Expose
    private long id;

    public Key_ getKey() {
        return key;
    }

    public void setKey(Key_ key) {
        this.key = key;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/talents/specializationdata/PlayableClass.kt
package com.BlizzardArmory.model.warcraft.talents.specializationdata

import com.BlizzardArmory.model.common.Key
import com.google.gson.annotations.SerializedName


/**
 * The type Playable class.
 */
data class PlayableClass(

    @SerializedName("key")
    var key: Key,

    @SerializedName("name")
    var name: String,

    @SerializedName("id")
    var id: Long

)